tools/wptrunner/wptrunner/testdriver-extra.js

```diff
@@ -233,99 +282,99 @@
-        return create_action("click_fedcm_dialog_button", {dialog_button, context});
+        return create_context_action("click_fedcm_dialog_button", context, {dialog_button});
     };
 
     window.test_driver_internal.select_fedcm_account = function(account_index, context = null) {
-        return create_action("select_fedcm_account", {account_index, context});
+        return create_context_action("select_fedcm_account", context, {account_index});
     };
 
     window.test_driver_internal.get_fedcm_account_list = function(context = null) {
-        return create_action("get_fedcm_account_list", {context});
+        return create_context_action("get_fedcm_account_list", context, {});
     };
 
     window.test_driver_internal.get_fedcm_dialog_title = function(context = null) {
-        return create_action("get_fedcm_dialog_title", {context});
+        return create_context_action("get_fedcm_dialog_title", context, {});
     };
 
     window.test_driver_internal.get_fedcm_dialog_type = function(context = null) {
-        return create_action("get_fedcm_dialog_type", {context});
+        return create_context_action("get_fedcm_dialog_type", context, {});
     };
 
     window.test_driver_internal.set_fedcm_delay_enabled = function(enabled, context = null) {
-        return create_action("set_fedcm_delay_enabled", {enabled, context});
+        return create_context_action("set_fedcm_delay_enabled", context, {enabled});
     };
 
     window.test_driver_internal.reset_fedcm_cooldown = function(context = null) {
-        return create_action("reset_fedcm_cooldown", {context});
+        return create_context_action("reset_fedcm_cooldown", context, {});
     };
 
     window.test_driver_internal.create_virtual_sensor = function(sensor_type, sensor_params={}, context=null) {
-        return create_action("create_virtual_sensor", {sensor_type, sensor_params, context});
+        return create_context_action("create_virtual_sensor", context, {sensor_type, sensor_params});
     };
 
     window.test_driver_internal.update_virtual_sensor = function(sensor_type, reading, context=null) {
-        return create_action("update_virtual_sensor", {sensor_type, reading, context});
+        return create_context_action("update_virtual_sensor", context, {sensor_type, reading});
     };
 
     window.test_driver_internal.remove_virtual_sensor = function(sensor_type, context=null) {
-        return create_action("remove_virtual_sensor", {sensor_type, context});
+        return create_context_action("remove_virtual_sensor", context, {sensor_type});
     };
 
     window.test_driver_internal.get_virtual_sensor_information = function(sensor_type, context=null) {
-        return create_action("get_virtual_sensor_information", {sensor_type, context});
+        return create_context_action("get_virtual_sensor_information", context, {sensor_type});
     };
 
     window.test_driver_internal.set_device_posture = function(posture, context=null) {
```

Comment:
Do the last two command here not need the same `create_context_action` change as everything else?